sceneview/src/main/java/io/github/sceneview/model/GLTFLoader.kt

```diff
@@ -0,0 +1,107 @@
+package io.github.sceneview.model
+
+import android.content.ContentResolver
+import android.content.Context
+import android.net.Uri
+import androidx.lifecycle.LifecycleCoroutineScope
+import com.google.android.filament.gltfio.FilamentAsset
+import com.google.ar.sceneform.rendering.ModelRenderable
+import io.github.sceneview.Filament
+import io.github.sceneview.utils.fileBuffer
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.future.await
+import kotlinx.coroutines.withContext
+import java.nio.Buffer
+import java.nio.ByteBuffer
+
+object GLTFLoader {
```

Comment:
Do you think we should separate the two loaders and let the developers select the right loader?